tools/ci/taskcluster-run.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
```

Comment:
So, I think I'm OK with landing this as-is, but I wonder what the effect would be of moving the logic in this file into `wpt run` directly? If one added `--commit-range` as an argument to that function then the only things that wouldn't directly fit would be getting the arguments right per-browser and gzipping artifacts. Those could perhaps be moved out into the task definitions.